.github/workflows/release.yml

```diff
@@ -37,6 +37,9 @@ jobs:
         run: |
           sudo xcode-select -switch /Applications/Xcode_12.app
 
+      - name: Carthage Bootstrap
+        run: carthage bootstrap --use-xcframeworks
```

Comment:
This is for `AmplitiudeCore` which does use XCFrameworks since each platform supports all architectures which does not work with a regular fat framework. This option tells carthage to build `AmplitudeCore` into an xcframework rather than just a regular framework.